folly/Portability.h

```diff
@@ -440,18 +440,6 @@ constexpr auto kCpplibVer = 0;
 #endif
 } // namespace folly
 
-// Define FOLLY_USE_CPP14_CONSTEXPR to be true if the compiler's C++14
-// constexpr support is "good enough".
-#ifndef FOLLY_USE_CPP14_CONSTEXPR
-#if defined(__clang__)
-#define FOLLY_USE_CPP14_CONSTEXPR __cplusplus >= 201300L
-#elif defined(__GNUC__)
-#define FOLLY_USE_CPP14_CONSTEXPR __cplusplus >= 201304L
-#else
-#define FOLLY_USE_CPP14_CONSTEXPR 0 // MSVC?
-#endif
-#endif
-
 #if FOLLY_USE_CPP14_CONSTEXPR
```

Comment:
Need to address this too in the same PR.

Reply:
Ah, good point. I'll fix up `FOLLY_CPP14_CONSTEXPR` and also `FOLLY_STORAGE_CPP14_CONSTEXPR`. 

`FOLLY_STORAGE_CPP14_CONSTEXPR` may break ICC. Do we make any guarantees in supporting a min version of ICC? See a8b6dd79.

Reply:
I've fixed this up now. Please take a peek.